roles/edpm_extra_mounts/meta/argument_specs.yml

```diff
@@ -0,0 +1,33 @@
+---
+argument_specs:
+  # ./roles/edpm_extra_mounts/tasks/main.yml entry point
+  main:
+    short_description: The main entry point for the edpm_extra_mounts role.
+    description: Multiple lines description
+    options:
+      edpm_extra_mounts:
+        type: list
+        default: []
+        description: |
+          A list of additional nfs mounts to be added to the edpm compute node.
```

Comment:
nit: A list of additional mounts (e.g. for nfs) to be added ...

Reply:
done